### PR TITLE
Update TV example app clusters

### PR DESCRIPTION
**Pull request by lazarkov**
#### Problem
TV example app does not support TV clusters.
Clusters have received an update in the spec

#### Summary of Changes
- Updated all TV example app clusters that will be used by the application
- Updated the xml files of cluster to follow minor spec changes
- Updated chip tool zap, to enable sending of CHIP commands through chip tool
- Updated chip controller zap, to enable sending of CHIP commands through chip tool

#### Note
Everything which is not supported by the CHIP framework currently is noted with TODO comment.

#### Test
- Tested locally using chip-tool client and newly created tv-server
- Used the `./scripts/tools/zap_regen_all.py ` and `./gn_build.sh ` to verify the building is successful

### CODE REVIEW

File: examples/chip-tool/BUILD.gn [hunk 1]
```diff
@@ -21,7 +21,7 @@ import("${chip_root}/src/app/chip_data_model.gni")
 assert(chip_build_tools)
 
 chip_data_model("data_model") {
-  cluster_sources = [ "media-playback-client" ]
+  cluster_sources = [ ]
```
Comment (lazarkov) [inline examples/chip-tool/BUILD.gn:24]: There's no need for media-playback-client callback commands.

File: src/controller/python/BUILD.gn [hunk 2]
```diff
@@ -32,7 +32,7 @@ config("controller_wno_deprecate") {
 }
 
 chip_data_model("python_data_model") {
-  cluster_sources = [ "media-playback-client" ]
+  cluster_sources = [ ]
```
Comment (lazarkov) [inline src/controller/python/BUILD.gn:35]: There's no need for media-playback-client code.

File: src/darwin/Framework/CHIP/BUILD.gn [hunk 3]
```diff
@@ -60,7 +60,7 @@ static_library("framework") {
 }
 
 chip_data_model("darwin_data_model") {
-  cluster_sources = [ "media-playback-client" ]
+  cluster_sources = [ ]
```
Comment (lazarkov) [inline src/darwin/Framework/CHIP/BUILD.gn:63]: There's no need for media-playback-client code

File: examples/chip-tool/BUILD.gn [hunk 4]
```diff
@@ -21,7 +21,7 @@ import("${chip_root}/src/app/chip_data_model.gni")
 assert(chip_build_tools)
 
 chip_data_model("data_model") {
-  cluster_sources = [ "media-playback-client" ]
+  cluster_sources = [ "thermostat-client" ]
```
Comment (woody-apple) [inline examples/chip-tool/BUILD.gn:24]: Is this just a bad diff? I assume we don't mean to replace this?

**Review by woody-apple — CHANGES_REQUESTED** [review]

File: examples/chip-tool/BUILD.gn (hunk 4 above)
Comment (lazarkov) [reply examples/chip-tool/BUILD.gn:24]: No it's not I want to replace it. There's no need for media playback client code as it's just sending response. 
What I haven't knew at the time is that if you name your cluster commads properly you get that for free.
Server command name: **Play**
Client command name: **Play**Response

If every client command has the Response string attached to a name you don't need client code

**Review by woody-apple — APPROVED** [review]

**Review by jelderton — APPROVED** [review]

**Review by chrisdecenzo — APPROVED** [review]
Looks like mostly auto-get code, hence the large number of files

**Pull request merged**